packages/devtools_app/test/test_infra/test_data/memory/heap/heap_graph_mock.dart

```diff
@@ -0,0 +1,193 @@
+// Copyright 2024 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:typed_data';
+
+import 'package:collection/collection.dart';
+import 'package:devtools_app/src/shared/memory/class_name.dart';
+import 'package:vm_service/vm_service.dart';
+
+typedef RefsByIndex = Map<int, List<int>>;
+typedef ClassByIndex = Map<int, HeapClassName>;
+
+class HeapSnapshotGraphMock implements HeapSnapshotGraph {
```

Comment:
rename this to "TestHeapSnapshotGraph". We should reserve the name "mock" for true mocks.